makefile

```diff
@@ -88,25 +90,25 @@ $(path_sw2lib)/$(lib_sw2):
 # Note: `-I..` is required for `#include "ST_defines.h"`
 # in SOILWAT2 headers that are included by SOILWAT2 code
 	@(cd $(path_sw2) && $(MAKE) lib \
-		CC="$(CC)" CPPFLAGS="$(CPPFLAGS) -I.." CFLAGS="$(CFLAGS)" AR="$(AR)" \
+		CC="$(CC)" CPPFLAGS="$(CPPFLAGS) -I.." CFLAGS="$(CFLAGS) $(std_c99)" AR="$(AR)" \
```

Comment:
compiling the SOILWAT2 library with `... CFLAGS="$(CFLAGS) $(std_c99)" ...` expands to `... -std=c99 ... -std=c11 ...`
--> fix by replacing `CFLAGS="$(CFLAGS) $(std_c99)"` with `CFLAGS="$(CFLAGS)" set_std="$(std_c99)"`